### DISCUSSION TITLE
Does not use the the inputted host to connect DB

### DISCUSSION THREAD

**Discussion by kexiuru** [dbeaver/dbeaver#16162] [Q&A]
I'm using DBeaver to connect remote MySQL, and found DBeaver does not use my inputted host to connect but another host.
![image](https://user-images.githubusercontent.com/17047719/153688916-64c23406-652c-49fd-a506-f6f93ed82535.png)

**Comment by flambchap** [comment]
Hi @kexiuru. MySQL's account names consist of two parts separated by the `@` sign. The part on the right is a hostname. MySQL's documentation states:

>The server performs matching of host values in account names against the client host using the value returned by the system DNS resolver for the client hostname or IP address.

In other words, the hostname is an allowed host from which the user can connect. Therefore, it has nothing to do with the Server Host.
